Golangci-lint on mac-os times out often

**Describe the bug**
Pre-checkin Action Go / Golangci-lint (macos-latest) has been very flaky recently with the following error:

``` 
Run golangci-lint5m 32s
##[error]Process completed with exit code 2.
Run make golangci
===> Installing Golangci-lint <===
golangci/golangci-lint info checking GitHub for tag 'v1.21.0'
golangci/golangci-lint info found version: 1.21.0 for v1.21.0/darwin/amd64
golangci/golangci-lint info installed .golangci-bin/golangci-lint
level=error msg="Timeout exceeded: try increase it by passing --timeout option"
make: *** [golangci] Error 4
##[error]Process completed with exit code 2.
```
Status board https://github.com/vmware-tanzu/antrea/actions?query=workflow%3AGo
Most recent failures are related to this issue

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -1,7 +1,7 @@
 # golangci-lint configuration used for CI
 run:
   tests: true
-  timeout: 5m
+  timeout: 10m
   skip-files:
     - ".*\\.pb\\.go"
   skip-dirs-use-default: true

